### PR TITLE
adding search bar and removing dupilcate the list

### LINKED ISSUE #17: adding searchbar in the view page

### PR DESCRIPTION
**Pull request by mariyareshma**

### CODE REVIEW

File: lib/view/home_page.dart [hunk 1]
```diff
@@ -59,11 +60,30 @@ class HomePageState extends State<HomePage> {
     super.initState();
   }
 
+  List<String> getSuggestion() {
+    var suggestion = <String>[];
+    for (var person in allPersons) {
+      if (suggestion.contains(person.name) == false) {
```
Comment (rajkumar82) [inline lib/view/home_page.dart]: modify 
if(suggestion.contains(person.name)== false)   
to 
if(!suggestion.contains(person.name))

File: lib/view/home_page.dart [hunk 2]
```diff
@@ -59,11 +60,30 @@ class HomePageState extends State<HomePage> {
...
+        suggestion.add(person.name!);
+      }
+    }
+    return suggestion;
```
Comment (rajkumar82) [inline lib/view/home_page.dart]: also rename suggestion to suggestions

**Review by rajkumar82 — CHANGES_REQUESTED** [review]
added 2 comments, fix them

**Comment by mariyareshma** [comment]
modify the home page

**Comment by mariyareshma** [comment]
modify the changes in the view page

**Review by rajkumar82 — APPROVED** [review]

**Pull request merged**